### DISCUSSION TITLE
Unable to Perform SELECT After Schema Deletion and Restoration in Newly Created Database Account

### DISCUSSION THREAD

**Discussion by LeeKangMo** [dbeaver/dbeaver#34358] [Q&A]
After deleting a schema and restoring the database to a previous state, CRUD permissions were assigned. However, when querying information_schema.role_table_grants, the granted permissions are not being retrieved, despite being successfully assigned. What could be the reason for this issue?

**Comment by TB-HunBoka** [comment]
For me, its sounds like a database specific question. Can you describe what DB are you talking about? (MySQL or PostgreSQL)
It would be also useful for someone with more experience to help you to answer:
Did you use any DBeaver specific function?
How did you restore the database? Like, is there a script you used, a built-in DB function. Is this rollback executed in a specific order?
Did you manually assign roles after the roll back, or it was part of the roll back?
Can you assign roles manually and then these new roles are retrieved successfully?
